src/course-outline/utils.jsx

```diff
@@ -110,7 +110,11 @@ const getHighlightsFormValues = (currentHighlights) => {
 };
 
 const scrollToElement = (ref) => {
-  ref.current?.scrollIntoView({ behavior: 'smooth' });
+  ref.current?.scrollIntoView({
```

Comment:
Do we want this to scroll even if the element is already in view? When I have a section in the center of my screen and I add a new subsection, the page scrolls so that the new subsection is at the bottom of my screen and the add button is no longer visible.

Reply:
@KristinAoki Updated to only scroll of element is not visible.